### PR TITLE
update go-da to version v0.0.1

### PR DESCRIPTION
**Pull request by hoank101**
## Overview

Update version go-data to v0.0.1, this version work compatible with cosmos-sdk-0.50.x

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

## Summary by CodeRabbit

- **Refactor**
  - Updated the submission function for improved data handling and additional configuration options.

### CODE REVIEW

**Comment by hoank101** [comment]
hi @tuxcanfly @gupadhyaya can you review PR?

**Comment by tuxcanfly** [comment]
@hoank101 can you please rebase on main?

File: celestia/celestia.go [hunk 1]
```diff
@@ -74,7 +74,7 @@ func (c *CelestiaDA) Commit(daBlobs []da.Blob) ([]da.Commitment, error) {
 }
 
 // Submit submits the Blobs to Data Availability layer.
-func (c *CelestiaDA) Submit(daBlobs []da.Blob) ([]da.ID, []da.Proof, error) {
+func (c *CelestiaDA) Submit(daBlobs [][]byte, options *da.SubmitOptions) ([][]byte, [][]byte, error) {
```
Comment (tuxcanfly) [inline celestia/celestia.go:77]: The mock `Submit` calls will need to be updated as well.

**Review by tuxcanfly — CHANGES_REQUESTED** [review]

File: celestia/celestia.go (hunk 1 above)
Comment (hoank101) [reply celestia/celestia.go:77]: I just fixed it

**Comment by tuxcanfly** [comment]
@hoank101 Thanks for PR, we decided to go with an alternative upgrade path see #46. Will have to close this unfortunately, but hope we can collaborate on other PRs!

**Pull request closed without merging**